.github/workflows/test.yml

```
@@ -0,0 +1,21 @@
+name: Test
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - '**' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)